services/movie_session.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import Movie, CinemaHall, MovieSession
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int
+                         ) -> None:
+    MovieSession.objects.create(show_time=movie_show_time,
+                                cinema_hall=cinema_hall_id,
+                                movie_id=movie_id
+                                )
+
+def get_movies_sessions(session_date: str = None
+                        ) -> QuerySet:
```

Comment:
The return type should be `QuerySet[MovieSession]` instead of just `QuerySet` to specify the type of objects in the queryset.